src/qibocal/update.py

```diff
@@ -0,0 +1,28 @@
+"""Helper functions to update parameters in platform."""
+from qibolab.platform import Platform
+from qibolab.qubits import QubitId
+
+from qibocal.protocols.characterization.utils import GHZ_TO_HZ
+
+
+def update_readout_frequency(results: dict[QubitId, float], platform: Platform):
+    """Update readout frequency value in platform for each qubit in results."""
+    for qubit, freq in results.items():
+        mz = platform.qubits[qubit].native_gates.MZ
+        freq_hz = int(freq * GHZ_TO_HZ)
+        mz.frequency = freq_hz
+        if mz.if_frequency is not None:
+            mz.if_frequency = freq_hz - platform.get_lo_readout_frequency(qubit)
```

Comment:
I would prefer
```suggestion
            mz.if_frequency = freq_hz - qubit.readout.lo_frequency
```
because the `platform.get_*`/`platform.set_*` will be deprecated at some point.

Reply:
Yes, I was planning to remove it. Thanks for catching this.